randovania/games/prime1/exporter/patch_data_factory.py

```diff
@@ -796,7 +796,7 @@ def component_number(name):
         seed = self.description.get_seed_for_player(self.players_config.player_index)
 
         boss_sizes = None
-        if self.configuration.random_boss_sizes:
+        if self.configuration.random_boss_sizes and all(v == 1.0 for v in [self.configuration.enemy_rando_range_scale_low, self.configuration.enemy_rando_range_scale_high]):
```

Comment:
Checking fields of config that no longer exist. It should also just check if the enemy-attributes exists